src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -347,7 +347,7 @@ PARAM_DEFINE_FLOAT(MC_YAWRAUTO_MAX, 45.0f);
  *
  * @unit deg/s
  * @min 0.0
- * @max 1000.0
+ * @max 1800.0
```

Comment:
Did you do experiements about the overshoot not accidentally breaching the 2000deg/s gyro limit barrier?

Reply:
No, not that high, but I did go over 1000. Would be interesting to know indeed.
Looks like others are actually going to the 2000 limit: https://github.com/betaflight/betaflight-configurator/issues/997